examples/grpc/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "npm_package_bin")
+
+npm_package_bin(
+    name = "grpc",
+    outs = [
+        "api_grpc_pb.d.ts",
```

Comment:
All these out files could be inffered from the name of proto if you were to make tiny macro wrapper